firmware/arduino/chessboard/uart.ino

```diff
@@ -191,6 +203,14 @@ bool makeMove(volatile char * message)
   return true;
 }
 
+void sendParamsToPi(volatile char currentState, volatile char extraByte, volatile char moveCount)
+{
+  sentMessage[0] = currentState;
+  sentMessage[1] = extraByte;
+  sentMessage[2] = moveCount;
+  sendMessageToPi(sentMessage);
+}
```

Comment:
I don't think this needs to be a new function, can simply set the values of `sentMessage` in `sendMessageToPi`